### ISSUE TITLE
Vaporware?

### ISSUE THREAD

**Issue by PowerPress** [r3dsh1ft/RedzSIGINT#1]
Has the source to this project gone Vaporware? Shame it seemed really impressive.

**Comment by iamredshift** [comment]
Great question.
The real answer to that is no, I just been trying to push out a few reports
for work and not have really spent a lot of time focusing on this but I'll
make sure to get something out for you this week. Code wise I gotta have
stuff up there tomorrow just the documentation part that's taking the
longest.

I can email you when I put up the code tomorrow or the day next but not all
the documentation is going to be up there still just waiting on that to get
through various forms of QA.

Hope this email makes sense I'm using voice to text for it, if it does not
this email me back when I'm in front of my computer again I'll type out
what it should have said.

Have a great day,
DP

**Comment by HEADHUNTE** [comment]
since it's been almost 3 years since it's inception of your surveillance Jeep project.  I am interested in its design and asking for any updates to your software and also a schematic wiring diagram you used in putting your hardware together please. Mr. Porter I also am in Colorado possible a local neighbor. Thank you for responding or contact me direct info@e-bailbonds.us or (303) 350-6351

**Comment by P3A-CE2017** [comment]
4 years....

Anything that can be done to help?

**Comment by HEADHUNTE** [comment]
Who is this please?

 

Rev. Dr. C.M. Schmidt, Jr., Th.D., D.D.

e-Bailbonds

(303) 350-6351

**Comment by PowerPress** [comment]
This project will never be released and was never planned to be released.
It was a stunt to get publicity nothing more.

**Comment by iamredshift** [comment]
The project was planed on being released. 
There was a few items which brought it to a halt. 
The biggest one was my research and development laptop, and some SDRs were stolen from me a few years ago (a little while after I made the first set of GIT commits), and that really deflated my want to continue on work which would not bring in money. 
If this project was a publicity stunt, it certainly was the worst publicity stunt in the world. I made no money from it, and if it was a publicity stunt I would have taken up the offers from a few conferences to bring the Jeep to their conferences and speak on it, but I had work to do and now employees to pay so I prioritized them over rebuilding research that lead to no profit. 

Check the GIT this week for updates on how to build this system in your own vehicle. 
Software wise, I will be slowly rebuilding the research and pushing things out. But as this does not help make my employee's payroll, it is not the top item on my priority list.

@PowerPress let me know how you want to help and we can sync up via a call to get things rolling at a more steady rate.
